### PR TITLE
Support all datatypes

### PR DESCRIPTION
**Pull request by mickohanlon23**
Switch the display value of the reference that is appended to the questionnaire based upon the resource type of the incoming FHIR resource. 

Within the mCODE IG, most resource types that we could expect to be passed in from the CQL will have a CodeableConcept type within its `.code` attribute. The exceptions are that a [`Specimen`](http://hl7.org/fhir/us/mcode/StructureDefinition-mcode-genetic-specimen.html#profile) will have a CodeableConcept within its `.type` attribute, and a [`MedicationStatement`](http://hl7.org/fhir/us/mcode/StructureDefinition-mcode-cancer-related-medication-statement.html) will have a CodeableConcept within its `.medicationCodeableConcept` attribute.

I also increased functionality so that the display value can either come from the `text` attribute of the Codeable Concept, or from the `display` value within the Coding of the Codeable Concept (in the mCODE IG these are the same value, but neither of them are required so I figured I would check for both). If none are present, the display value defaults to the reference.

### CODE REVIEW

File: src/utils/results-processing.ts [hunk 1]
```diff
@@ -157,3 +159,19 @@ function getExpressionName(item: R4.IQuestionnaire_Item): string | undefined {
...
   return undefined;
 }
+
+function getDisplayValue(fhirResource: any, resourceType: string): string | undefined {
+  let attribute: string;
+  // Determine which Codeable Concept attribute to query based upon incoming resource type
+  switch (resourceType) {
+    case 'Specimen':
+      attribute = 'type';
+      break;
```
Comment (pmadria) [inline src/utils/results-processing.ts:169]: Looks good - medicationStatement needs to be added as well? Another case for `medicationCodeableConcept`

Comment (mickohanlon23) [reply src/utils/results-processing.ts:169]: Whoops sorry, forgot to commit that part. Should be there now.

**Comment by mgramigna** [comment]
There are a bunch of commits that are already merged into master on this branch (guessing it was branched off of my data type addition once that has since been merged).

Is this properly rebased?

**Comment by mickohanlon23** [comment]
> There are a bunch of commits that are already merged into master on this branch (guessing it was branched off of my data type addition once that has since been merged).
> 
> Is this properly rebased?

I just merged locally and pushed back up. It looks like it now incorporates all of the recent changes.

**Comment by pmadria** [comment]
After rebase - the PR is good and the functionality works as intended.

**Review by pmadria — APPROVED** [review]

**Review by mgramigna — APPROVED** [review]

**Pull request merged**